content/commands/httl/index.md

```diff
@@ -62,14 +62,14 @@ See also the [`HPTTL`]({{< relref "/commands/hpttl" >}}) command that returns th
 ## Example
 
 ```
-redis> HTTL no-key 10 3 field1 field2 field3
+redis> HTTL no-key 10 FIELDS 3 field1 field2 field3
```

Comment:
Should the "10" be there after "no-key"? Doesn't seem consistent with the syntax spec and the second HTTL command line lower down.
```suggestion
redis> HTTL no-key FIELDS 3 field1 field2 field3
```

Reply:
Thanks for catching this, @andy-stark-redis . Fixed in subsequent push.